packages/inference/src/snippets/python.ts

```diff
@@ -100,16 +98,58 @@ const snippetGenerator = (templateName: string, inputPreparationFn?: InputPrepar
 		providerModelId?: string,
 		opts?: Record<string, unknown>
 	): InferenceSnippet[] => {
+		/// Hacky: hard-code conversational templates here
+		if (
+			model.pipeline_tag &&
+			["text-generation", "image-text-to-text"].includes(model.pipeline_tag) &&
+			model.tags.includes("conversational")
+		) {
+			templateName = opts?.streaming ? "conversationalStream" : "conversational";
+			inputPreparationFn = prepareConversationalInput;
+		}
```

Comment:
I guess another solution would be to have the caller pass a `isConversational: boolean` flag or similar - wdyt?